nrich-notification/README.md

```diff
@@ -4,15 +4,28 @@
 
 ## Overview
 
-`nrich-notification` is a module intended for creation of notifications on server side which are shown on the client side. It supports info (i.e. 'Entity has been saved'), warning (i.e. 'Validation
-failed') and error (i.e. 'Exception occurred') notification severity levels, and can also send a list of validation errors to client side along with original notification. Notification messages are
-resolved by message keys from Spring's `MessageSource`. Keys can be either fixed strings or in case of exceptions their class names.
+`nrich-notification` is a module intended for addition of notifications into the server-side response which can be shown on the client-side.
+It's main purpose is to provide a unified response format.
+
+It supports three different notification severity levels which are described in table bellow.
+
+| Notification severity | Description                                                                                     |
+|:---------------------:|-------------------------------------------------------------------------------------------------|
+|        `INFO`         | information that an action was successfully executed such as 'Entity has been saved' or similar |
```

Comment:
Maybe `an action was successfully...`,  `some validation has failed...` and `an exception occurred...` (without repeating `information that` prefix).

Additionally, not sure if I would use backticks for severity levels inside this table.

Reply:
It does look better with backtiks, I asked a couple of my collegues what they think and they all agreed that it looks good to them. Also, changed the table.